lib/src/chart/bar_chart/bar_chart_data.dart

```diff
@@ -358,13 +370,19 @@ class BarChartRodData with EquatableMixin {
   /// ```
   BarChartRodData({
     double y,
-    Color color,
+    List<Color> colors,
+    Alignment gradientFrom,
```

Comment:
We have implemented gradient before in the LineChart,
And we have used [Offset](https://api.flutter.dev/flutter/dart-ui/Offset-class.html) for `gradientFrom`, and `gradientTo` properties, for the sake of consistency, it is better to use Offset object here too.

Also, please check how we handled gradient before, you can start by reading [here](https://github.com/imaNNeoFighT/fl_chart/blob/master/lib/src/chart/line_chart/line_chart_painter.dart#L539).
Thanks!